parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -79,7 +88,12 @@ downloadGPUDrivers() {
   apt_get_update
   retrycmd 30 5 60 curl -fLS https://us.download.nvidia.com/tesla/$GPU_DV/NVIDIA-Linux-x86_64-${GPU_DV}.run -o ${GPU_DEST}/nvidia-drivers-${GPU_DV} || exit 85
   tmpDir=$GPU_DEST/tmp
-  apt_get_download 20 30 libnvidia-container1=1.6.0* libnvidia-container-tools=1.6.0* nvidia-container-toolkit=1.6.0* nvidia-container-runtime=3.6.0* || exit 85
+  for apt_package in $NVIDIA_PACKAGES; do
+    apt_get_download 20 30 "${apt_package}=${NVIDIA_CONTAINER_TOOLKIT_VER}*" || exit 85
+    cp ${APT_CACHE_DIR}${apt_package}_${NVIDIA_CONTAINER_TOOLKIT_VER}* $PERMANENT_CACHE_DIR || exit 85
```

Comment:
If this were to use `cp -al` it would copy it to the permanent cache directory but via a hard link such that you don't double the space used.
(Assuming these two are on the same filesystem volume, which today they are.)  It also makes sure that they have the same permissions as the original.

Reply:
But we need to copy it because we need to not depend upon the files in the apt cache directory being cleaned up.

Reply:
nevermind, hard link, understood